### PR TITLE
Fix iPad sidebar

### PR DESCRIPTION
**Pull request by Sjmarf**
Sidebar navigation wasn't working at all

### CODE REVIEW

File: Mlem/App/Views/Root/Tabs/Feeds/SubscriptionListItemView.swift [hunk 1]
```diff
@@ -19,9 +19,14 @@ struct SubscriptionListItemView: View {
...
-            HStack(spacing: 15, content: label)
-        }
+        MultiplatformView(phone: {
+            NavigationLink(.community(community), label: label)
+        }, pad: {
+            Button(action: {
+                navigation.root = .community(community)
+            }, label: label)
+                .buttonStyle(EmptyButtonStyle())
+        })
```
Comment (EricBAndrews) [inline Mlem/App/Views/Root/Tabs/Feeds/SubscriptionListItemView.swift]: Can this use the `multiplatformButton` defined in `SubscriptionListView`? (extracted to its own full view obv)

**Review by EricBAndrews — CHANGES_REQUESTED** [review]
Nice! Works perfectly. The list items probably want rounded corners + dividers:
Screenshot 2024-09-19 at 6 08 18 PM

**Comment by Sjmarf** [comment]
> Nice! Works perfectly. The list items probably want rounded corners + dividers:

This was there before this PR, I'm leaving as OOS and fixing in #1326

File: Mlem/App/Views/Root/Tabs/Feeds/SubscriptionListItemView.swift (hunk 1 above)
Comment (Sjmarf) [reply Mlem/App/Views/Root/Tabs/Feeds/SubscriptionListItemView.swift]: Sure

**Review by EricBAndrews — APPROVED** [review]
LGTM! 🚀

**Pull request merged**